src/components/AppCatalog/AppList/AppListSearch.js

```diff
@@ -1,18 +1,41 @@
+import styled from '@emotion/styled';
 import PropTypes from 'prop-types';
 import React from 'react';
 
+const Form = styled('form')`
+  width: 300px;
+  font-size: 14px;
+  position: relative;
+
+  input {
```

Comment:
I remember us discussing that styled components are preferred over nested selectors. Can't see anything written in `fmt` about it. What's your opinion on that?

Reply:
I think it's ok to start off nested and only extract them when it seems necessary. What are the drawbacks of using nested selectors?

Reply:
In this case there shouldn't be any problem. The problems arrive when we have nested components, and with selectors we'll also apply styles to the elements inside those nested components.
So, if we ever add another component (let's call it `ResetButton`), which would have a nested `<i/>` element inside it, we'll actually apply these styles:
```
color: #fff;
font-size: 13px;
font-weight: 800;
```
to the `<i/>` element inside the `ResetButton` component.

Reply:
Ah ok. unless you have strong opinion about it, Im ok with potentially running into that and then fixing it when it's necessary. What do you think?

Reply:
Yeah, let's move on 👍

Reply:
The `fmt` section in question is stuck in this PR: https://github.com/giantswarm/fmt/pull/118